### PR TITLE
SONAR-8438 New web service api/system/ping

### PR DESCRIPTION
**Pull request by simonbrandhof**
SONAR-8438 New web service api/system/ping

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/platform/ws/PingAction.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import static org.apache.commons.io.IOUtils.write;
+
+public class PingAction implements SystemWsAction {
+  @Override
+  public void define(WebService.NewController controller) {
+    controller.createAction("ping")
+      .setDescription("Answers \"pong\" as plain-text")
+      .setSince("6.3")
+      .setResponseExample(getClass().getResource("example-ping.txt"))
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ws/PingAction.java]: In order for the example to be more visible, we're trying to start the file by the name of the action -> ping-example.txt
(I know that it's not done everywhere yet...)

Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ws/PingAction.java]: ok I didn't know. I will fix but still I suggest you to do a mass change.

Comment (simonbrandhof) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ws/PingAction.java]: fixed in next fixup commit

Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/platform/ws/PingAction.java]: Perfect ! You're right we should do some mass update

**Pull request closed without merging**